jekyll/_cci2/security.md

```diff
@@ -18,7 +18,7 @@ Security is our top priority at CircleCI, we are proactive and we act on securit
 ## Encryption
 CircleCI uses HTTPS or SSH for all networking in and out of our service including from the browser to our services application, from the services application to your builder fleet, from our builder fleet to your source control system, and all other points of communication. In short, none of your code or data travels to or from CircleCI without being encrypted unless you have code in your builds that does so at your discretion. Operators may also choose to go around our SSL configuration or not use TLS for communicating with underlying systems.
 
-The nature of CircleCI is that our software has access to your code and whatever data that code interacts with. All jobs on CircleCI run in a sandbox (specifically, a Docker container or on-demand VM container) that stands alone from all other builds and is not accessible from the Internet or from your own network. The build container pulls code via git over SSH. Your particular test suite or job configurations may call out to external services or integration points within your network, and the response from such calls will be pulled into your jobs and used by your code at your discretion. After a job is complete the container that ran the job is destroyed and rebuilt. All environment variables you store inside CircleCI are encrypted at rest and sent to your job's containers using SSH.
+The nature of CircleCI is that our software has access to your code and whatever data that code interacts with. All jobs on CircleCI run in a sandbox (specifically, a Docker container or on-demand VM container) that stands alone from all other builds and is not accessible from the Internet or from your own network. The build container pulls code via git over SSH. Your particular test suite or job configurations may call out to external services or integration points within your network, and the response from such calls will be pulled into your jobs and used by your code at your discretion. After a job is complete the container that ran the job is destroyed and rebuilt. All environment variables are stored using [Hashicorp Vault](https://www.vaultproject.io/). Environment variables are encrypted using AES256-GCM96 and are unavailable to CircleCI employees.
```

Comment:
s/stored/encrypted/

Single word change but important for accuracy.

Detailed explanation for internal understanding:
Vault does secrets management in a number of different ways.  It can store secrets directly, be an encryption service, integrate with aws security features, generate one-time-passwords, etc.  We are looking to leverage different features of Vault for different CCI features.

For user environment variables, we use the Vault encryption service (they call it the "transit secret store").  It's technically possible to "store customer environment variables in vault", but we chose this approach because Postgres is better at scaling for large volumes of data.

Thus, we "encrypt environment variables using vault" and store the resulting ciphertext in postgres.

Reply:
@tito-villalobos Is that Postgres database in GPC or AWS?